circle.yml

```diff
@@ -26,7 +28,7 @@ dependencies:
       export PATH="$HOME/miniconda/bin:$PATH" &&
       source activate test &&
       conda install ciocheck -c spyder-ide &&
-      conda install spyder &&
+      pip install git+https://github.com/spyder-ide/spyder@3.x &&
```

Comment:
@jitseniesen, this doesn't work because Spyder is not using setuptools, so it won't pull its dependencies.

Please take a look at what we do in `spyder-notebook`'s circle.yml to workaround this.